src/invoices/InvoiceDetails/InvoiceBatchVoucherExport/InvoiceBatchVoucherExport.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Col,
+  KeyValue,
+  Row,
+} from '@folio/stripes/components';
+import {
+  FolioFormattedTime,
+} from '@folio/stripes-acq-components';
+
+import { BATCH_VOUCHER_EXPORT_STATUS_LABEL } from '../../../common/constants';
+import ExportVoucherButton from '../../../settings/BatchGroupConfigurationSettings/ExportVoucherButton';
+import BatchGroupValue from '../BatchGroupValue';
+
+const InvoiceBatchVoucherExport = ({ batchVoucherExport, exportFormat }) => {
+  return (
+    <Row>
+      <Col xs={4}>
+        <BatchGroupValue
+          id={batchVoucherExport.batchGroupId}
+          label={<FormattedMessage id="ui-invoice.invoice.details.batchVoucherExport.batchGroup" />}
+        />
+      </Col>
+
+      <Col xs={4}>
+        <KeyValue
+          label={<FormattedMessage id="ui-invoice.invoice.details.batchVoucherExport.fileName" />}
+        >
+          <FolioFormattedTime dateString={batchVoucherExport.metadata.createdDate} />
+          <ExportVoucherButton
+            batchVoucherId={batchVoucherExport.batchVoucherId}
+            fileName={batchVoucherExport.metadata.createdDate}
```

Comment:
Are you sure that back-end sends the right date format? I doubt it

Reply:
BE sends data `2020-08-07T07:23:33.528+0000` that converts to `2020-08-07T07_23_33_528+0000`